Documentation/Books/Manual/Indexing/Geo.md

```diff
@@ -296,7 +299,9 @@ With Holes:
 
 - The exterior ring should not self-intersect.
 - The interior rings must be contained in the outer ring
-- Interior rings cannot overlap (or touch) with each other
+- Rings cannot share edges, they may however share vertices
+- Polygon rings should follow the right-hand rule for orientation
+  (counterclockwise external rings, clockwise internal rings).
 
```

Comment:
- The exterior ring should not self-intersect.
 - The interior rings must be contained in the outer ring
 - No two rings can cross each other, i.e. no ring may intersect both the
   interior and exterior face of another ring
 - Rings cannot share edges, they may however share vertices
 - No ring may be empty
 - Polygon rings should follow the right-hand rule for orientation
   (counterclockwise external rings, clockwise internal rings).

Reply:
are adding these changes ?

Reply:
nevermind